### PR TITLE
Browse products

### PR DESCRIPTION
**Pull request by mcpenchel**
Hey, I think I'm finished. Can you take a look, @juliaf1 ?

### CODE REVIEW

File: app/models/product.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Product < ApplicationRecord
+
+    def pretty_info
+        "#{title} at #{price}"
```
Comment (mcpenchel) [inline app/models/product.rb:4]: Hmmmm do you think my pretty_info method should have more stuff?

Comment (juliaf1) [reply app/models/product.rb:4]: Yes, I do

**Comment by juliaf1** [comment]
Where is the browsing products part? Doesn't look complete, Penchel...

**Comment by juliaf1** [comment]
Nice

**Pull request merged**